layer_factory/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.10.2)
 if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-    add_definitions(-DVK_USE_PLATFORM_WIN32_KHR -DVK_USE_PLATFORM_WIN32_KHX -DWIN32_LEAN_AND_MEAN)
+    add_definitions(-DVK_USE_PLATFORM_WIN32_KHR -DVK_USE_PLATFORM_WIN32_KHX -DWIN32_LEAN_AND_MEAN -DNOMINMAX)
```

Comment:
I strongly recommend preferring `target_compile_definitions` to `add_definitions`. This can get really ugly as other projects are included and/or other projects consume this one.

Reply:
Agreed, I am just keeping the fix to minimum to meet SDK release deadline...